.github/CODEOWNERS

```diff
@@ -0,0 +1,21 @@
+#####################################################
```

Comment:
Do know if that can be a source of conflict but there is already a CODEOWNERS file at the root of this project.

Reply:
oh, thanks! I didn't  realize there is one (and didn't know it could be in root). Fixing